source/help/messaging/formatting-text.rst

```diff
@@ -125,23 +125,44 @@ Renders as:
 Code Block
 ----------
 
-Create a code block by indenting each line by four spaces, or by placing ``````` on the line above and below your code.
+Creating a fixed width code block is recommended for pasting multi-line blocks of code or other text output that is easier to read with fixed width font alignment. Examples include block text snippets, ASCII tables, log files, etc.
 
-Example:
+Renders as:
 
 .. code-block:: none
 
+  this is my code
+  123 alignment
+  alignment 456
+  fixed width!
 
-  ```
-  code block
-  ```
+This can be accomplished in one of two ways:
 
-Renders as:
+ 1. Indenting each line by four spaces
+
+Example:
 
 .. code-block:: none
 
+      this is my
+      code block
+
+  ^^^^ 4x spaces
+
 
+ 2. Placing 3x backtics (\`\`\`) on the line directly above and directly below your code
+
+Example:
+
+.. code-block:: none
+
+  ```
+  this is my
   code block
+  ```
+
+
+* **TIP**! - If you're having trouble as a first-timer, assuming default keyboard shortcuts for most clients, this can be accomplished by entering: 3x backticks <\`\`\`>, <SHIFT>+<ENTER>, <type_your_code>, <SHIFT>+<ENTER>, and finally 3x backticks <\`\`\`>.
```

Comment:
Let's use the sphinx formatting for this, then it will render in a special "Tip" box:
```
.. tip ::
If you're new to Markdown, assuming default keyboard shortcuts for most clients, this can be accomplished by entering: 3x backticks <\`\`\`>, <SHIFT>+<ENTER>, <type_your_code>, <SHIFT>+<ENTER>, followed by 3x backticks <\`\`\`>.
```

Reply:
this didn't work ... I can't see any such facility as per http://www.sphinx-doc.org/en/stable/markup/inline.html

Reply:
I also tried fuddling with the `kbd` syntax ... but upon make of html it complains as an unknown role... http://www.sphinx-doc.org/en/stable/markup/inline.html#role-kbd

Reply:
Ah! got _something_ to work well; Figured out the :kbd: role properly (so now the keyboard sequences look much cleaner, as well as I got _something_ for the TIP suggestion; it's possibly not what you were intending ... but it's something? (if you have the exact syntax or commit the fix feel free)

... incoming commit

Reply:
9b9e3d3..4742673